config/35-pilot.ini

```
@@ -0,0 +1,39 @@
+;===================================================================
+;                          Pilots
+;===================================================================
+
+; For each pilot type, add a new pilot section.
+; Each pilot name must be unique for the entire grid, so make sure to not
+; pick anything generic like "MAIN".
+
+; This data is used to determine where OSG pilot jobs can be sent, so it's
+; important to keep it up to date.
+
+
+;[Pilot PILOT_NAME]
+;; The number of cores for this pilot type.
+;cpucount = 1
+;; The amount of memory (in megabytes) for this pilot type.
+;ram_mb = 2500
+;; This is a whole node pilot; cpucount and ram_mb are ignored if this is true.
+;whole_node = false
+;; The number of GPUs available
+;gpucount = 0
+;; The maximum number of pilots of this type that can be sent
+;max_pilots = CHANGEME
+;; The maximum wall-clock time a job is allowed to run for this pilot type,
+;; in minutes.
+;max_wall_time = 1440
+;; The queue which jobs should be submitted to in order to run on this resource.
+;; Equivalent to the HTCondor grid universe classad attribute "remote_queue"
+;queue =
+;; True if the pilot should require singularity on the workers.
+;require_singularity = true
+;; The OS of the workers; allowed values are "rhel6", "rhel7", "rhel8", or "ubuntu18".
+;; This is required unless require_singularity = true
+;os = rhel7
+;; Send test glideins?
```

```suggestion
;; Send test pilots?
```

This is the only place you don't use `pilot` in the config